<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="DESKTOP-3MLN4MN" name="com.qa.tests.ExtendedSicknessBenefitCompleteFlow" tests="15" failures="0" timestamp="21 Oct 2021 10:59:55 GMT" time="331.781" errors="2">
  <testcase name="tearDown" time="0.005" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.tests.ExtendedSicknessBenefitCompleteFlow.tearDown(ExtendedSicknessBenefitCompleteFlow.java:787)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- tearDown -->
  <testcase name="Login" time="12.176" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="ESBMed8Certificate" time="23.010" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="ContinuationOfPreviousSpellCertificateForESB1" time="40.266" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="ContinuationOfPreviousSpellCertificateForESB2" time="19.527" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="ContinuationOfPreviousSpellCertificateForESB3" time="28.298" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="ESBCreateClaim" time="18.428" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="ESBP" time="97.863" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="BMLoginforESB" time="33.248" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="ESBProcessClaimBMLogin2" time="56.916" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow"/>
  <testcase name="MRLoginforESB" time="2.044" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow">
    <error type="org.openqa.selenium.UnhandledAlertException" message="Unexpected modal dialog (text: Intimation to the MR is Successful !): Intimation to the MR is Successful !
Build info: version: &#039;2.53.0&#039;, revision: &#039;35ae25b1534ae328c771e0856c93e187490ca824&#039;, time: &#039;2016-03-15 10:43:46&#039;
System info: host: &#039;DESKTOP-3MLN4MN&#039;, ip: &#039;10.30.11.76&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_281&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=40.0.3, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 8412b4dc-83c9-49d0-a830-3af151051167">
      <![CDATA[org.openqa.selenium.UnhandledAlertException: Unexpected modal dialog (text: Intimation to the MR is Successful !): Intimation to the MR is Successful !
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'DESKTOP-3MLN4MN', ip: '10.30.11.76', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_281'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=40.0.3, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 8412b4dc-83c9-49d0-a830-3af151051167
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
at org.openqa.selenium.remote.ErrorHandler.createUnhandledAlertException(ErrorHandler.java:187)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:154)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:316)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteNavigation.to(RemoteWebDriver.java:969)
at com.qa.tests.ExtendedSicknessBenefitCompleteFlow.MRLoginforESB(ExtendedSicknessBenefitCompleteFlow.java:611)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- MRLoginforESB -->
  <testcase name="ESBProcessClaimMRLogin" time="0.000" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow">
    <skipped/>
  </testcase> <!-- ESBProcessClaimMRLogin -->
  <testcase name="BMLoginforESBRateCard" time="0.000" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow">
    <skipped/>
  </testcase> <!-- BMLoginforESBRateCard -->
  <testcase name="ESBProcessClaimRateCardBMLogin2" time="0.000" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow">
    <skipped/>
  </testcase> <!-- ESBProcessClaimRateCardBMLogin2 -->
  <testcase name="ESBProcessClaimBMLogin3" time="0.000" classname="com.qa.tests.ExtendedSicknessBenefitCompleteFlow">
    <skipped/>
  </testcase> <!-- ESBProcessClaimBMLogin3 -->
</testsuite> <!-- com.qa.tests.ExtendedSicknessBenefitCompleteFlow -->
